.github/workflows/terraform-cognito-test.yaml

```diff
@@ -1,3 +1,4 @@
+# todo: test how this will work with the dual input provider changes
```

Comment:
remove? we will see how it goes anyway 😄

Reply:
haha ok

Reply:
removed